scripts/add_binary_version_header.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
```

Comment:
This script doesn't **add** version headers; it replaces them. Rename accordingly.

Reply:
How about adding a subdirectory like this: `scripts/version_header/replace_version.py` ?

Due to its super vague name, the `scripts/` directory is unsurprisingly becoming a dumpsite of bits totally unrelated to each other.

If these scripts are only for testing then they should even be moved to `tests/kernel/version_header/`